RELEASE/scripts/autoscend/iotms/clan.ash

```diff
@@ -428,6 +427,22 @@ boolean zataraClanmate(string who)
 #	}
 #	set_property("_clanFortuneConsultUses", get_property("_clanFortuneConsultUses").to_int() + 1);
 
+	int attempts = 0;
+	int player = 3038166;
+	string consultOverrideName = get_property("auto_consultChoice");
+	string name = get_player_name(player);
+	if (consultOverrideName != "")
```

Comment:
Also check `&& is_online(consultOverrideName)`

Reply:
I feel like if someone has explicitly set an override, you shouldn't try to fall back to a default one, even if the override is offline.

Reply:
That's fair